bin/cron/update_project_count

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+#
+#
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/only_one'
+require 'cdo/properties'
+require src_dir 'database'
+require_relative '../../deployment.rb'
+require 'cdo/honeybadger'
+
+# Production database has a global max query execution timeout setting.  This 20 minute setting can be used
+# to override the timeout for a specific session or query.
+MAX_EXECUTION_TIME = 1_200_000
+MAX_EXECUTION_TIME_SEC = MAX_EXECUTION_TIME / 1000
+
+# Connection to read from Pegasus reporting database.
+PEGASUS_REPORTING_DB_READER = sequel_connect(
+  CDO.pegasus_reporting_db_reader,
+  CDO.pegasus_reporting_db_reader,
+  query_timeout: MAX_EXECUTION_TIME_SEC
+)
+
+# This default number of lines of code is from 2019-1-21.
+DEFAULT_NUMBER_OF_PROJECTS = 35_000_000
+
+NUMBER_PROJECTS_SIGNIFICANT_DECREASE = 10_000
+
+def main
+  # Get the current amount
+  project_count = PEGASUS_REPORTING_DB_READER[:storage_apps].where(standalone: true).count
+
+  # check if number is bigger than previous count
+  existing_project_count = Properties.get(:metrics)['project_count'] || 0
+  if (existing_project_count - project_count) >= NUMBER_PROJECTS_SIGNIFICANT_DECREASE
+    # Log Honeybadger error that the project count somehow went down
+    error_message = "storage_apps standalone project count decreased from #{existing_project_count} to #{project_count}"
+    raise error_message
+  end
+
+  # save value for use
+  time = DateTime.now
+  Properties.set :metrics, {
+    created_at:               time,
+    created_on:               time.to_date,
+    petition_signatures:      Properties.get(:metrics)['petition_signatures'],
```

Comment:
Similar comment about a comment here.  It would be nice if we had an implementation of `Properties.set` that could just update certain JSON fields but let's not worry about it for this PR.